lib/models/user-model.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema(
+    {
```

Comment:
You don't need to necessarily add _id in the schema.. that is auto-generated by mongodb

Reply:
I wanted to use a readable _id but it's okay. I am going to use the autogenerated one.